app/util.js

```diff
@@ -108,19 +108,19 @@ const searchword = location => {
         if (word && url) {
             const q = location.replace(`${word} `, "")
             if (q && location.startsWith(`${word} `)) {
-                const queries = q.split(",").map(item => item.trim())
+                const queries = q.split(",")
                 let urlString = url
                 let counter = 1
                 const patternMatches = (urlString.match(/%s/g) || []).length
                 while (urlString.includes("%s") && counter < patternMatches) {
-                    if (0 in queries) {
-                        urlString = urlString.replace(/%s/, encodeURIComponent(queries.shift()))
+                    if (queries[0]) {
+                        urlString = urlString.replace(/%s/, encodeURIComponent(queries.shift().trim()))
                     } else {
                         urlString = urlString.replace(/%s/g, "")
                     }
                     counter += 1
                 }
-                const remainderString = queries.join(", ")
+                const remainderString = queries.join(",").trim()
```

Comment:
This gets trimmed here to avoid this behavior:
`ghrepo Jelmerro, Vieb`
being replaced to:
`github.com/Jelmerro/ Vieb`

Side effect is that the case in which there's only one %s will also loose the initial spaces...

I'd be ok with that but maybe someone really needs their spaces at the begginning of their searches...